db/seeds.rb

```diff
@@ -4,9 +4,9 @@
 
 Item.create(name: "hot chocolate", image: "adkjf.png",
             category_id: 1, description: "amazazing",
-            purchase_url: "www.dfjhs.com")
+            purchase_url: "www.dfjhs.com", user_id: 1)
 Item.create(name: "cold chocolate", image: "asdfdkjf.png",
-            category_id: 1, description: "not")
+            category_id: 1, description: "not", user_id: 1)
```

Comment:
We can't guarantee that there will be a user with id of 1. Grab or create a user object and put it here.